web/beacon-app/src/features/auth/types/RegisterService.ts

```diff
@@ -25,7 +25,10 @@ export type NewUserAccount = Omit<
   'user_id' | 'name' | 'organization' | 'domain' | 'terms_agreement' | 'privacy_agreement'
 >;
 
-export type NewInvitedUserAccount = Omit<User, 'user_id' | 'organization' | 'domain'>;
+export type NewInvitedUserAccount = Omit<
```

Comment:
I didn't change the existing `User` type since the BE hasn't been updated yet. This will likely be updated though.